target/rspamd/local.d/redis.conf

```diff
@@ -0,0 +1,6 @@
+# documentation: https://rspamd.com/doc/configuration/redis.html
+
+servers = "127.0.0.1:7788";
```

Comment:
This is the redis port? It differs from supervisor config?

Reply:
Ooooops, wrong port :D